source_handlers.go

```diff
@@ -331,3 +331,50 @@ func SourceCheckAvailability(c echo.Context) error {
 
 	return c.JSON(http.StatusAccepted, map[string]interface{}{})
 }
+
+// SourcesRhcConneectionList returns all the connections related to a source.
+func SourcesRhcConneectionList(c echo.Context) error {
+	paramId := c.Param("source_id")
+
+	sourceId, err := strconv.ParseInt(paramId, 10, 64)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, util.ErrorDoc("invalid id provided ", "400"))
+	}
+
+	filters, err := getFilters(c)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, util.ErrorDoc(err.Error(), "400"))
+	}
+
+	limit, offset, err := getLimitAndOffset(c)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, util.ErrorDoc(err.Error(), "400"))
+	}
+
+	// Check if the given source exists.
+	sourceDao, _ := getSourceDao(c)
```

Comment:
Not sure we want to ignore the error and then check the previous error again :P

Reply:
Good catch! Solved.